drivers/cc110x/cc110x_communication.c

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2018 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup   drivers_cc110x
+ * @{
+ *
+ * @file
+ * @brief     Functions to communicate with the CC1100/CC1101 transceiver
```

Comment:
As far as I tell, most drivers call this file `_internal.c`. Any reason to call it `_communication.c`?

Reply:
The internal functions of the driver are split among multiple files. This file only contains the communication related internal functions. Other internal stuff is e.g. placed in `cc110x_rxtx.c` and so on. So I'd argue that `cc110x_communication.c` is more precisely stating what is inside the fine